buildout-mercator.cfg

```diff
@@ -10,6 +10,11 @@ frontend.static_dir = src/mercator/mercator/build
 frontend_package_name = mercator
 backend_package_name = adhocracy_mercator
 
+[thentos_captcha]
+required = true
+backend_url = http://localhost:6542/
+frontend_url = http://localhost:6542/
+
```

Comment:
i don't understand.  the captcha microservice runs two listeners: one that is contacted by the a3 backend, and one contacted by the a3 frontend.  both look different in each installation, similarly to adhocracy's own frontend_url, backend_url, web sockets url etc.

if you don't want to make them configurable here, how do you suggest a3 frontend and backend will find out how to contact thentos-captcha?